Microsoft.Toolkit.Uwp.Services/Services/Twitter/TwitterOAuthRequest.cs

```diff
@@ -58,17 +59,17 @@ internal class TwitterOAuthRequest
         /// <returns>awaitable task</returns>
         public async Task ExecuteGetStreamAsync(Uri requestUri, TwitterOAuthTokens tokens, TwitterStreamCallbacks.RawJsonCallback callback)
         {
-            using (var request = new HttpHelperRequest(requestUri, HttpMethod.Get))
+            using (var request = new HttpRequestMessage(HttpMethod.Get, requestUri))
             {
                 var requestBuilder = new TwitterOAuthRequestBuilder(requestUri, tokens);
 
-                request.Headers.Authorization = HttpCredentialsHeaderValue.Parse(requestBuilder.AuthorizationHeader);
+                request.Headers.Authorization = AuthenticationHeaderValue.Parse(requestBuilder.AuthorizationHeader);
 
-                using (var response = await HttpHelper.Instance.GetInputStreamAsync(request).ConfigureAwait(false))
+                using (var response = await client.SendAsync(request).ConfigureAwait(false))
```

Comment:
instead of normal request this requests stream like call (from what I remember hence the additional method in HttpHelper
https://docs.microsoft.com/en-gb/dotnet/api/system.net.http.httpclient.getstreamasync?view=netstandard-1.4 might be what you need